src/test/java/com/epam/tc/hw2/ex2/Ex2.java

```diff
@@ -0,0 +1,168 @@
+package com.epam.tc.hw2.ex2;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+
+public class Ex2 {
+
```

Comment:
Class names should be nouns, in mixed case
with the first letter of each internal word capitalized. Try to keep your class names simple
and descriptive. Use whole words—avoid
acronyms and abbreviations (unless the abbreviation is much more widely used than the
long form, such as URL or HTML).

From https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
Fixed a bit. Code divided into separate logical classes and methods in the following work.